pages/docs/open-source-roadmap/what-next.js

```diff
@@ -17,7 +17,7 @@ export default function DocsNext() {
         <h1 className="text-4xl mb-4 font-bold">
           Open Source Roadmap: What next?
         </h1>
-        <p>Coming soon</p>
+<p>It does'nt stop there! You can become a maintainer. If you want to do this for the communities you're in, you can do things like identifying duplicate issues or spam pull requests, aiding contributors with their pull requests, giving contributors friendly reminders about the Contributing Guide for first-time contributors. You can even take what you have learned and start your own Open Source project. The possibilities are endless.</p>
```

Comment:
There are two problems 🙂 it says `does'nt` but should say `doesn't` but because react doesn't like single quotes it should be `doesn&apos;t`. Hopefully that helps.